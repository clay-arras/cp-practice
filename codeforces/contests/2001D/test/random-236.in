1
8
24 25 15 7 23 23 1 18
