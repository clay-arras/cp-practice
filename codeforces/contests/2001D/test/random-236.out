7
24 25 15 7 23 1 18
